dojo/templates/dojo/findings_list_snippet.html

```diff
@@ -219,6 +222,7 @@ <h3 class="has-filters">
                             <th class="nowrap">{% dojo_sort request 'Product' 'test__engagement__product__name'%}</th>
                             {% endif %}
                             <th>Service</th>
+                            <th>Planned Remediation Date</th>
```

Comment:
The list has already a lot of fields and this title needs much more space than the actual date. Can we just write "Planned Remediation"?

Reply:
Fine with me!